tests/test_binary.nim

```diff
@@ -1008,6 +1012,285 @@ proc testsForCompletion(binaryPath: string) =
         outp.contains(&"Please choose a shell. For example: `configlet completion -s bash`")
         exitCode == 1
 
+proc testsForFmt(binaryPath: static string) =
+  const formattedTrackDir = testsDir / ".test_nim_track_repo"
+  const unformattedTrackDir = testsDir / ".test_elixir_track_repo"
+
+  # Setup: clone track repo, and checkout a known state (formatted)
+  setupExercismRepo("nim", formattedTrackDir,
+                    "ea91acb3edb6c7bc05dd3b050c0a566be6c3329e") # 2022-01-22
+
+  # Setup: clone track repo, and checkout a known state (unformatted)
+  setupExercismRepo("elixir", unformattedTrackDir,
+                    "07448c4f870c15f8191196a2b01e8bf09708b8ce") # 2022-01-11
```

Comment:
I'm using both the Elixir and Nim repo here, with Nim being the one where all the config files (except the track `config.json` file) are formatted, and Elixir having none of the formatted.